Command/DumpCommand.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+namespace Becklyn\MysqlDumpBundle\Command;
+
+use Doctrine\DBAL\Connection;
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Helper\FormatterHelper;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Question\ConfirmationQuestion;
+use Symfony\Component\Security\Core\Exception\AccessDeniedException;
+
+/**
+ * Handles the importing of the images from the command line
+ */
+class DumpCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure ()
+    {
+        $this
+            ->setName('becklyn:db:dump')
+            ->setDescription("Dumps the configured or given databases using 'mysqldump' to .sql files.")
+            ->addOption(
+                'connections',
+                'c',
+                InputOption::VALUE_REQUIRED,
+                'Dumps the provided database connections. Multiple connection names are separated by <info>comma (,)</info>',
+                null
+            )
+            ->addOption(
+                'path',
+                'p',
+                InputOption::VALUE_REQUIRED,
+                'The folder path where the .sql file will be saved. Defaults to %kernel.root%/var/db_backups/ if not specified otherwise.',
```

Comment:
`"The folder path where the .sql file will be saved. Defaults to '%kernel.root%/var/db_backups/'.",`